sources/packages/api/src/utilities/date-utils.ts

```diff
@@ -115,6 +115,18 @@ export function getDateOnlyFromFormat(
   return getDateOnly(isoDate);
 }
 
+/**
+ * Get the date only part of a date/time object.
+ * @param date date/time to have the date extracted.
+ * @returns date only string in ISO format YYYY-MM-DD.
+ */
+export function getDateOnlyString(date?: Date): string | null {
```

Comment:
getDateOnlyFormat method returns the format YYYY MM DD, can we please rename the method name to something that defines the ISO format? like getISODateOnlyString? or something?

Reply:
Method renamed.